src/app/Library/CrudPanel/Traits/AutoSet.php

```diff
@@ -219,9 +221,9 @@ public function getDbColumnsNames()
         if (! $this->driverIsSql()) {
             $columns = $fillable;
         } else {
+            $schema = new ModelSchema($this->model);
             // Automatically-set columns should be both in the database, and in the $fillable variable on the Eloquent Model
-            $columns = $this->model->getConnection()->getSchemaBuilder()->getColumnListing($this->model->getTable());
-
+            $columns = $schema->getColumnsNames();
```

Comment:
@pxpm are we 100% sure this will return the exact same thing? Otherwise we have to qualify this as a breaking change, you know... 👀 Maybe it makes sense to add a unit test for this one? Or is it already covered by existing unit tests?

Reply:
Yeah, tests were not failing locally dunno why but I see the difference. `lowercase` vs `camelCase`. 

I've pushed a fix to it. 

This is the test:
```php
public function testGetDbColumnsNames()
    {
        $this->crudPanel->setModel(ColumnType::class);

        $columnNames = $this->crudPanel->getDbColumnsNames();

        $this->assertEquals(array_keys($this->expectedColumnTypes), $columnNames);
    }
```
No breaking changes here. 

Cheers